Replace static Cytoscape plugin with npm package

itisAliRH opened the pull request:
This PR replaces the `cytospace` visualisation static files with the npm package ([@itisalirh/galaxy-cytoscape](https://www.npmjs.com/package/@itisalirh/galaxy-cytoscape)) developed using [@guerler/galaxy-charts-starter](https://github.com/guerler/galaxy-charts-starter). Also removed duplicate <data> elements for "Depth First Search" in the XML file.

## How to test the changes?
(Select all options that apply)
- [ ] I've included appropriate [automated tests](https://docs.galaxyproject.org/en/latest/dev/writing_tests.html).
- [x] This is a refactoring of components with existing test coverage.
- [ ] Instructions for manual testing are as follows:
  1. [add testing steps and prerequisites here if you didn't write automated tests covering all your changes]

## License
- [x] I agree to license these and all my past contributions to the core galaxy codebase under the [MIT license](https://opensource.org/licenses/MIT).

itisAliRH committed:
Remove static Cytoscape visualization plugin build and related files

itisAliRH committed:
Add Cytoscape visualization npm package @itisAliRH/galaxy-cytoscape

guerler commented on config/plugins/visualizations/cytoscape/config/cytoscape.xml:
```diff
@@ -13,7 +13,10 @@
     </params>
-    <entry_point entry_point_type="chart" src="script.js"/>
+    <requirements>
+        <requirement type="npm" version="latest" package="@itisalirh/galaxy-cytoscape"/>
```
Please specify an actual version here.

itisAliRH committed:
Update Cytoscape npm requirement to specific version

Refined the npm package version for Cytoscape to 3.30.3-2

itisAliRH committed:
Refactor Cytoscape config for clarity and simplicity

Removed unnecessary options and streamlined the edge type selection to a boolean format.

itisAliRH committed:
Update npm package version for cytoscape plugin to 3.30.3-3

guerler committed:
Use npm package from galaxyproject

guerler reviewed (approved):
Package has been moved to @galaxyproject as per request of @mvdbeek. Thank you  @itisAliRH.

bgruening committed:
Merge branch 'dev' into cytoscape-npm-package
